projects/templates/projects/view_project.html

```diff
@@ -46,7 +46,21 @@ <h2>
       </tr>
       <tr>
         <th>Type</th>
-        <td>{{ project.type }}</td>
+        <td>
+          {% if is_admin %}
+            <span id='project_type'>{{ project_type|default_if_none:'' }}</span><span id='icon_show_type_edit' title="Edit" style="padding-left: 5px;" class="glyphicon glyphicon-edit"></span>
+            <span id='project_type_edit_form' class='hidden'>
```

Comment:
You could probably set the ID of the `<form>` to `project_type_edit_form` and put the hidden class on that as well, then you don't need this `<span>`, I think?

Reply:
that's for hiding the form and two buttons - "update" and "cancel" buttons.